### ISSUE TITLE
Version1

### ISSUE THREAD

**Issue by iamsermie** [penoutk/Calculator#1]
@penoutk @fansi96 
we cant do two operations at the same time, and i think the calculations should be in the label field

**Comment by fansi96** [comment]
yeah , i agree on that, the result should be displayed in the lower input textbox.
